tantalus/migrations/0101_auto_20190122_2303.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
can you rebase this and rename the migration, one has been added so now 0101 is taken and this needs to be 0102 referencing 0101, otherwise good

Reply:
Will do